app/controllers/movies_controller.rb

```diff
@@ -0,0 +1,14 @@
+class MoviesController < ApplicationController
+  def show
+    @movie = Movie.find(params[:id])
+    @actors = @movie.actors_sorted_by_age
+    @average_age = @movie.average_actor_age
+  end
+    
+    
+  def update
+    movie = Movie.find(params[:id])
+    movie.movie_actors.create(actor_id: params[:actor_id])
```

Comment:
Interestingly, this action is called `#update`, but we're only doing a `.create` action inside. While the code you wrote fulfills the requirements for the user story, this qualifies more as a `create` action for the `MovieActor` model, so maybe it should go in that controller.